backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/utils/Modifier.kt

```diff
@@ -53,3 +59,15 @@ internal fun Modifier.autoMirror(enabled: Boolean = true): Modifier =
       this@drawWithContent.drawContent()
     }
   }
+
+internal fun Modifier.unboundClickable(role: Role? = null, onClick: () -> Unit): Modifier =
```

Comment:
What about doing clickable which mimics the Compose clickable but adds bounded param?